musicbot/config.py

```diff
@@ -918,6 +961,14 @@ def run_checks(self) -> None:
         if self.enable_local_media and not self.media_file_dir.is_dir():
             self.media_file_dir.mkdir(exist_ok=True)
 
+        if self.cookies_path.is_file():
+            log.warning(
+                "Cookies TXT file detected. MusicBot will pass them to yt-dlp.\n"
```

Comment:
I'd like a warning here about using personal daily accounts

Reply:
I've updated the warning text here and added a sleep to make sure folks see it.